Program.cs

```diff
@@ -1,12 +1,261 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using static Exercises.Program;
+﻿namespace Exercises
+{
+    public static class ExercisesOOP
+    {
+        private static void Main()
+        {
+            //Exercise 1;
+            Console.WriteLine("Exercise 1.");
+
+            var person = new Person();
+            person.SetSurname("Fleming");
+            person.SetName("Lise");
+            person.SetAge(33);
+
+            Console.WriteLine($"{person.GetSurname()} " +
+                $"{person.GetName()}, " +
+                $"{person.GetAge()}");
+
+            //Exercise 2;
+            Console.WriteLine("Exercise 2.");
+
+            var student = new Student();
```

Comment:
When we say we work in OOP, we must create a model that is already completely valid.
To do this, through the constructor we pass the parameters that are necessary for the model. Without them, the model makes no sense.

For example, a student can be without estates, without age?
If not, then create these arguments in the constructor.